wp-term-meta/wp-term-meta.php

```diff
@@ -48,13 +48,23 @@ private static function _post( $term_id, $taxonomy = null ) {
 
 		$post_query = new WP_Query( array(
 			'post_type' => self::post_type,
-			'post_status' => self::post_type,
+			// include draft status to prevent edge case with legacy data
+			// causing a replication bug
+			'post_status' => array( 'draft', self::post_type ),
```

Comment:
While this probably works I don't think I'm on board with this. It's not really addressing the root of the problem. 

I'm also pretty sure non-logged-in users will not be able to get post_status='draft' at all. Would much prefer a single step data clean-up (not in this code) and/or corrected code.